packages/components/header/README.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Header'
+type: 'component'
```

Comment:
Maybe they should be on the Layout section??
```suggestion
type: 'component'
section: 'layoutComponents'
```

Reply:
Makes sense - updating